listenbrainz/webserver/static/css/brainzplayer.less

```diff
@@ -162,6 +163,13 @@
 		}
 
 	}
+
+	.elapsed {
```

Comment:
I'd also add `flex-grow: 0;`so that it doesn't take more space than required.
That'll be important in smaller sizes:

Without:
<img width="458" alt="image" src="https://user-images.githubusercontent.com/6179856/169502400-af82bb0b-9b8f-4d07-8593-4b85e9daa3ce.png">

With:
<img width="458" alt="image" src="https://user-images.githubusercontent.com/6179856/169502468-ac03727b-e1d1-42e8-bc11-b2c49049a5cd.png">

Reply:
Noted.